push-fedora-40.sh

```diff
@@ -3,28 +3,38 @@
 OS=fedora
 OSVER=fedora-40
 
-podman manifest create quay.io/foreman/nboci-files:$OSVER
+# create image index and set the annotation at the index level
+# TODO set the artifactType at the index level https://github.com/containers/podman/issues/23598
+podman manifest create quay.io/foreman/nboci-files:$OSVER --annotation org.pulpproject.netboot.version=1
 
+mkdir -p work
 rm -f ./work/*
```

Comment:
Would it make more sense to just delete the whole directory and then recreate it? That also removes any hidden files:
```suggestion
rm -rf ./work
mkdir work
```

And perhaps out of scope, but should each step that creates/needs files create their own subdirectory in `work`? Perhaps even with `mktemp -d -p work`. That way you only do a `mkdir -p "$WORKDIR"` at the start and and the end `rm -rf "$WORKDIR"`.

Reply:
In that case please remove the work directory from git, there is I think `.gitkeep` hidden file. I like the idea of creating a temporary directory and removing it afterwards.

Reply:
I can probably come up with a PR so should we merge this and do it as a follow up?

Reply:
Sure, let’s merge Ina just returned from longer leave and she might be busy. I am ready to do it as well if needed. This was just quick and dirty way of doing stuff, let’s fix that.

Reply:
https://github.com/theforeman/nboci-files/pull/2 is what I came up with. It escalated a bit, but individual commits should be reasonably easy to follow.